File: src/main/scala/CalcProtienMass.scala
package rosalind
import common._

object ProtienMass {
	def main(args: Array[String]) {
		val input_table = readLines("data/monoisotopic_table.txt")
		val input_list = input_table.map(s => s.split(' ')).toList
		println(input_list.map(_(0).length))
	}
}

File: src/main/scala/calcProtienMass.py
f = [l.split() for l in open("monoisotopic_table.txt").readlines()]
monoisotopic_table = dict([(k, float(v)) for [k, v] in f])
input_str = open("rosalind_prtm.txt").read().strip()
answer = sum([monoisotopic_table.get(c) for c in input_str])
print answer